### ISSUE TITLE
[TSDoc- Playground] : Heading hierarchy is not correct in Playground Page.

### ISSUE THREAD

**Issue by vagpt** [microsoft/rushstack-websites#252]
**Test Environment:**
Version: Version 24H2 (OS Build 26231.5000)
Browser Version: 127.0.2638.2 (Official build) dev (64-bit)
URL: https://tsdoc.org/play/
Tool: Accessibility Insight For Web

**Repro Steps:**
1. Open the above URL on the EDGE Dev browser.
2. Press the tab key and navigate to the 'PlayGround' option and press enter key to activate it.
3. Open the tool and turn on the heading toggle button on and observe the issue.

**Actual Result:**
On the Playground page, Heading hierarchy is not correct as all heading are defined as H1.

**Expected Result:**
On the playground page, heading hierarchy should be correct it should be h2.
Summary, Parameter and other heading should be defined in H2.

**User Impact:**
Screen reader users will not get the context of the page if they navigate in heading mode as incorrect heading level announcement will be confusing for users.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

Have feedback on bugs logged, please tag bug as “A11yRCA” and add your feedback in the comment section.

![Heading hierarchy is not correct in Playground Page](https://github.com/microsoft/rushstack-websites/assets/95913748/ce9f7ccb-c63b-473c-970b-cb8242c05ec7)
